### PR TITLE
feat: Better dataset site errors

### PR DESCRIPTION
**Pull request by nickevansuk**

### CODE REVIEW

File: packages/openactive-broker-microservice/app.js [hunk 1]
```diff
@@ -884,17 +891,29 @@ async function startPolling() {
 
   const harvesters = [];
 
+  const isParentFeed = {
```
Comment (civsiv) [inline packages/openactive-broker-microservice/app.js:894]: Totally optional as it doesn't affect functionality, just readability:

Calling this `isParentFeed` feels slightly misleading as it's being used as a 3-branch conditional below, and then just as a list of feeds in error statement.

Maybe slightly more readable would be:
```
const additionalTypeParentNonParentMap = {
https://openactive.io/SessionSeries': PARENT_ADDITIONAL_FEED,
....
```
and 
```
if (additionalTypeParentNonParentMap[dataDownload.additionalType] === PARENT_ADDITIONAL_FEED)
```
etc

imo `Object.keys(isParentFeed).map((x)` is a bit misleading as we're using it for all feeds, not just parent ones

**Review by civsiv — APPROVED** [review]

File: packages/openactive-broker-microservice/app.js (hunk 1 above)
Comment (nickevansuk) [reply packages/openactive-broker-microservice/app.js:894]: Oh sure - is `PARENT_ADDITIONAL_FEED` a `Symbol` here?

Comment (nickevansuk) [reply packages/openactive-broker-microservice/app.js:894]: (Merging now for speed)

Comment (civsiv) [reply packages/openactive-broker-microservice/app.js:894]: >Oh sure - is PARENT_ADDITIONAL_FEED a Symbol here?

Yeh exactly

**Pull request merged**